test/verify/check-connection

```diff
@@ -89,6 +89,32 @@ class TestConnection(MachineCase):
         b.click("#machine-reconnect")
         b.expect_load()
         b.enter_page("/system")
+        b.logout()
+
+        if not m.atomic_image:  # cannot write to /usr on Atomics
```

Comment:
We have `m.needs_writable_usr()` for these cases.

Reply:
Indeed, but we also don't actually have cockpit-session on the Atomic image. I updated the comment.

Reply:
Ack.